pom.xml

```diff
@@ -215,6 +215,11 @@
             <version>2.4.3</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-avro_2.12</artifactId>
+            <version>2.4.5</version>
```

Comment:
does <scope>provided</scope> work on your side (if the Spark environment provides it)? This reduces the size of the jar...

Reply:
Not exactly sure what you meant, but according to [this page](https://spark.apache.org/docs/latest/sql-data-sources-avro.html), "The `spark-avro` module is external and not included in `spark-submit` or `spark-shell` by default."

Reply:
Yes indeed, thank you for checking. Indeed then it is the right configuration.